woocommerce/woocommerce-ios
# Dynamic Dashboard: Error state for dashboard cards

Closes: #12546 


## Description

This PR handles the error state for dashboard cards. The same layout is applied for every card, and there are two actions available: contact support and retry.

## Testing instructions

- Turn off the internet connection on your device and build the app (assuming you are already logged in to a test store).
- Confirm that all cards show the loading state. 
- After all requests time out, confirm that all cards display the error state.
- Tap on "contact support" on any of the error state views, confirm that the support form is displayed.
- Tap on Retry button of any of the error state views, confirm that only the corresponding card is reloaded.
- Confirm that the old error banner is no longer displayed on the Dashboard screen when the Performance card fails to load data. 

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/abcabdff-a2da-46bf-8a38-3b89f3bbda63



---
